domain/storage/service/params.go

```diff
@@ -0,0 +1,26 @@
+// Copyright 2025 Canonical Ltd.
+// Licensed under the AGPLv3, see LICENCE file for details.
+
+package service
+
+import (
+	"github.com/juju/juju/core/storage"
+	internalstorage "github.com/juju/juju/internal/storage"
+)
+
+// ImportStorageParams contains the parameters for importing storage into a model.
+type ImportStorageParams struct {
+	// Kind is the kind of the storage to import.
+	Kind internalstorage.StorageKind
```

Comment:
Does it seem correct that we have this type/enum defined in an internal package?

Reply:
It never used to be internal and wasn't supported to be but it got moved by someone. 
We now need to go in and move a bunch of stuff out from internal storage and add it to core. This is one of the things.

Reply:
Ta would be nice to get this done in another PR